src/containers/my-quizzes/selectable-question-quiz-view/SelectableQuestionQuizView.tsx

```diff
@@ -38,15 +38,11 @@ const SelectableQuestionQuizView: FC<SelectableQuestionQuizViewProps> = ({
   })
 
   const onNext = () => {
-    if (!isLastQuestion) {
-      setSelectedIndex(selectedIndex + 1)
-    }
+    !isLastQuestion && setSelectedIndex(selectedIndex + 1)
   }
 
   const onBack = () => {
-    if (!isFirstQuestion) {
-      setSelectedIndex(selectedIndex - 1)
-    }
+    !isFirstQuestion && setSelectedIndex(selectedIndex - 1)
   }
```

Comment:
now, you can remove curly brackets from here

Reply:
ok, i will fix that